app/components/Request/messages.js

```diff
@@ -11,6 +11,8 @@ export default defineMessages({
   payment_request: 'Payment Request',
   total: 'Total',
   memo: 'Memo',
+  memo_help:
```

Comment:
When adding new strings you should run `yarn extract-messages` so that they get extracted into our translation files at `/app/translations`.

Also, if we are calling the element a `tooltip` then I think the translation key would be better named as `memo_tooltip` for consistency.

Reply:
Will do.